app/build.gradle

```diff
@@ -22,19 +23,32 @@ android {
         exclude 'META-INF/NOTICE'
         exclude 'META-INF/LICENSE'
     }
+
+    android {
+        lintOptions {
+            abortOnError false
```

Comment:
nope nope nope nope nope nope nope nope nope nope
nicht in diesem leben und nicht auf dieser welt